src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
The href attribute points to a '.scss' file, which is not directly usable by browsers. Ensure that this file is compiled to a '.css' file and link to that instead, or use a build tool to handle this automatically.